Lib/fontbakery/checkrunner.py

```diff
@@ -628,7 +644,7 @@ def check_order(self, order):
         raise ValueError(f'Order item {item} not found.')
     return order
 
-  def run(self, order=None):
+  def old_run(self, order=None):
```

Comment:
If the reasoning behing renaming this to `old_run` is simply to keep track of the old implementation, I'd suggest deleting it for good, as we have git commit history for that. Let's get rid of legacy/unused code instead of leaving it floating in the repo.

If there's other reason for that, I'd appreciate to have a comment placed here, right before this `old_run` method.

Reply:
Yes I'd definitely delete this for good. Just waitng for #2964

Reply:
`old_run` is gone in with 3faed57